pkg/operator/configobservation/auth/podsecurityadmission_test.go

```diff
@@ -25,78 +26,62 @@ func TestObservePodSecurityAdmissionEnforcement(t *testing.T) {
 	restrictedJSON, err := json.Marshal(restrictedMap)
 	require.NoError(t, err)
 
-	defaultFeatureSet := &configv1.FeatureGate{
-		Spec: configv1.FeatureGateSpec{
-			FeatureGateSelection: configv1.FeatureGateSelection{
-				FeatureSet:      "",
-				CustomNoUpgrade: nil,
-			},
-		},
-	}
+	defaultFeatureSet := featuregates.NewHardcodedFeatureGateAccess([]configv1.FeatureGateName{}, []configv1.FeatureGateName{})
 
-	corruptFeatureSet := &configv1.FeatureGate{
-		Spec: configv1.FeatureGateSpec{
-			FeatureGateSelection: configv1.FeatureGateSelection{
-				FeatureSet:      "Bad",
-				CustomNoUpgrade: nil,
-			},
-		},
-	}
+	// TODO provide a hardcoded test harness that allows not-ready and error
+	//corruptFeatureSet := &configv1.FeatureGate{
+	//	Spec: configv1.FeatureGateSpec{
+	//		FeatureGateSelection: configv1.FeatureGateSelection{
+	//			FeatureSet:      "Bad",
+	//			CustomNoUpgrade: nil,
+	//		},
+	//	},
+	//}
```

Comment:
The observer doesn't know about feature sets anymore. Any reason to keep this or the related test cases around and commented?